test/client/fixtures/runner/runner-test.js

```diff
@@ -64,32 +88,24 @@ asyncTest('Test iterator should not call Transport.fail twice (without screensho
 
     SETTINGS.TAKE_SCREENSHOTS_ON_FAILS = false;
 
-    var testRunner = new Runner();
-
     testRunner._onFatalError({ message: 'err1' });
     testRunner._onFatalError({ message: 'err2' });
 
     window.setTimeout(function () {
         equal(transportFailCount, 1);
 
-        SETTINGS.TAKE_SCREENSHOTS_ON_FAILS = savedTakeScreenshotOnFails;
-        transport.fatalError               = savedTransportFatalError;
-
         start();
     }, 100);
 });
 
 asyncTest('Test iterator should not call Transport.fail twice (with screenshots)', function () {
-    var savedTakeScreenshotOnFails = SETTINGS.TAKE_SCREENSHOTS_ON_FAILS;
-    var savedTransportFatalError   = transport.fatalError;
-    var transportFailCount         = 0;
+    var transportFailCount = 0;
 
-    transport.fatalError               = function () {
+    transport.fatalError = function () {
         transportFailCount++;
     };
-    SETTINGS.TAKE_SCREENSHOTS_ON_FAILS = true;
 
-    var testRunner = new Runner();
+    SETTINGS.TAKE_SCREENSHOTS_ON_FAILS = true;
```

Comment:
You have a discrepancy:
`SETTINGS.TAKE_SCREENSHOTS_ON_FAILS` here
`SETTINGS.TAKE_SCREENSHOT_ON_FAILS` above.
Please, correct